media/editor/virtualDocument.ts

```diff
@@ -414,7 +414,7 @@ export class VirtualDocument {
 			event.preventDefault();
 		} else if (new RegExp(/ArrowLeft|ArrowRight|ArrowUp|ArrowDown/gm).test(event.key)
 			|| ((event.key === "End" || event.key === "Home") && !event.ctrlKey)) {
-			this.navigateByKey(event.key, targetElement, event.shiftKey);
+			this.navigateByKey(event.key, targetElement, event.shiftKey, event);
```

Comment:
Would this be better as a meta key parameter. It feels weird that we extract a bunch of properties such as event.key and event.shiftKey, but then just decide to pass the whole event in anyways.

Reply:
Yeah this was kind of tacked onto the end because I don't really like how it works now. Ideally we're just pass the whole event into it and `navigateByKey` would handle it. But I think it's being used to without an event sometimes. I'll switch it to just pass in `metaKey`.